integration_tests/README.rst

```diff
@@ -3,13 +3,16 @@ How to run the tests in own environment
 
 Navigate to your Polar2Grid directory, then run::
 
+    conda env update -n jenkins_p2g_docs -f build_environment.yml
+    conda env update -n jenkins_p2g_docs -f jenkins_environment.yml
+    conda activate jenkins_p2g_docs
+    pip install .
+    conda env update -n jenkins_p2g_swbundle -f build_environment.yml
+    conda activate jenkins_p2g_swbundle
+    ./create_conda_software_bundle.sh polar2grid-YYYYMMDD-HHmmSS                                    # Creates a folder with the necessary scripts to run the tests
```

Comment:
You _could_ use ``datetime +%Y%m%d-%H%M%S`` (need the single back ticks) to place the date in the filename. Not necessary to change, but FYI. You may want to change the `MM` and `DD` in the above since they are the wrong case.

Reply:
Interesting, good to know. I have seen MM used for months also, but I will change this and keep it in mind for the future (https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html)!